connectivity/drivers/lora/TARGET_STM32WL/mbed_lib.json

```diff
@@ -47,6 +47,10 @@
         },
         "debug_tx": {
             "help": "GPIO pin for TX debug"
+        },
+        "debug_invert": {
```

Comment:
`debug_led_invert` might be better

Reply:
Absolutely the deal is that I wanted to be consistent with already related existing options. It took me some time to understand that ˋdebug_tx` was for led  and not a serial. So if I change it it would makes sense to change 2 other (and introduce breaking change). Whatever solution is fine for me

Reply:
> `debug_led_invert` might be better

Hi

Not sure, as primary goal of the debug GPIO is to easily use logic analyser for example.
Use f LED is only an "extension"

Reply:
> > `debug_led_invert` might be better
> 
> Hi
> 
> Not sure, as primary goal of the debug GPIO is to easily use logic analyser for example.
> Usage of LEDs is only an "extension"

makes sense, so how should I name this new config parameters, just let me know
`debug_led_invert` or keep consistency with `debug_invert`